app/main.py

```diff
@@ -1,8 +1,51 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: int,
+                 clean_mark: int,
+                 brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self,
+                 distance_from_city_center: float,
+                 clean_power: int, average_rating: float,
+                 count_of_ratings: int) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def wash_single_car(self, car: Car, clean_power: int) -> None:
+        if self.clean_power > car.clean_mark:
+            car.clean_mark = self.clean_power
+        else:
+            print("Wash station clean power is less or equals Car clean mark")
```

Comment:
The message printed here could be more informative. Consider including the actual values of `self.clean_power` and `car.clean_mark` in the message for better debugging and user feedback.